tests/preprocessor/special-macro-simple.slang.expected

```diff
@@ -0,0 +1,6 @@
+result code = 0
+standard error = {
+}
+standard output = {
+5 "tests/preprocessor/special-macro-simple.slang" 
```

Comment:
It is a minor point (and not something we need to rush to "fix" any time soon), but the `-E` mode for most C/C++ compilers produces preprocessed output with `#line` directives to ensure that file/line information is correct when the output is passed into a compiler again.

(I know it is weird the running a C preprocessor on a file results in a file that still has preprocessor directives, but `#line` is just special that way.)